src/components/MemberProgress/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { convertDate } from '../utilis';
+const MemberProgress = ({ currentMember: member }) => {
+  let id = 1;
+  return (
+    <div>
+      <div className='task-message'>{member.firstName + ' ' + member.lastName} progress:</div>
```

Comment:
create helper for fullName

Reply:
did hepler getFullName
```suggestion
export function getFullName(firstName, lastName) {
  return firstName + ' ' + lastName;
}
```